vendor/models/choice.py

```diff
@@ -307,3 +307,57 @@ class Country(models.IntegerChoices):
     ZA = 710, _("South Africa")
     ZM = 894, _("Zambia")
     ZW = 716, _("Zimbabwe")
+
+
+class StateChoices(models.TextChoices):
```

Comment:
I would change the class name to be more specific to the USA. Don't know if `UnitedStatesStateChoices` or `USAStateChoices`